src/protocol.h

```diff
@@ -418,12 +418,24 @@ class CInv
     std::string ToString() const;
 
     // Single-message helper methods
-    bool IsMsgTx()        const { return type == MSG_TX; }
-    bool IsMsgWtx()       const { return type == MSG_WTX; }
+    bool IsUndefined() const { return type == UNDEFINED; }
```

Comment:
nit: the methods `IsUndefined()` and `IsMsgWitnessTx()` are unused.

Reply:
Good point; probably worth removing them for now.

Reply:
Done